h2o-mapreduce-generic/build.gradle

```diff
@@ -17,6 +17,18 @@ dependencies {
     exclude module: "${defaultWebserverModule}"
   }
   compile project(":h2o-security")
+
+  //To make H2OTestRunner available
+  testCompile project(":h2o-core").sourceSets.test.output
```

Comment:
Agree this is very ugly, would rather move the required stuff into separate module